src/components/interactive-map.js

```diff
@@ -180,6 +194,16 @@ export default class InteractiveMap extends PureComponent {
     // Register additional event handlers for click and hover
     eventManager.on('mousemove', this._onMouseMove);
     eventManager.on('click', this._onMouseClick);
+
+    // run stub queued action
+    this._eventManager.queue.forEach(({events, ref}) => {
```

Comment:
If we really need a stub, Should we at least make this into a method on the `Stub`? Something like `EventManagerStub.transferToRealManager(eventManager)`?